miso-web/src/main/webapp/scripts/sample_ajax.js

```diff
@@ -709,18 +741,48 @@ Sample.ui = {
       break;
     }
   },
+
+  hideTissueProcessingFields: function() {
```

Comment:
showing/hiding tissueProcessing stuff in here probably isn't necessary, since it would be controlled in the JSP. This stuff is used when changing SampleClass, which you can only do when receiving (not propagating) a new sample